packages/testnet/contracts/Perpetual.sol

```diff
@@ -8,20 +8,20 @@ contract Perpetual {
 
     event LogForcedWithdrawalRequest(
```

Comment:
The solidity source code is the one place I would actually keep the original naming, because this will allow us to make sure that it works with those names.

Reply:
Makes sense.